.github/wiki/Home.md

```diff
@@ -26,34 +19,40 @@ How do we decide if a feature is **rejected**? 👎
 
 Let's have a look at some examples 🔎
 
-**Essential**:
+**Approved**:
 - [Timber](https://github.com/JakeWharton/timber): A logging library which is always used in all the projects we've worked on.
 - [Firebase App Distribution](https://firebase.google.com/docs/app-distribution): Continuous delivery always requires a boilerplate setup.
 - [Detekt](https://github.com/detekt/detekt): Code smell analysis is important to maintain code quality.
 - [Hilt](https://developer.android.com/training/dependency-injection/hilt-android): Dependency injection provides a good developer experience.
 - [Navigation Component](https://developer.android.com/guide/navigation/navigation-getting-started): A suite of libraries for in-app navigation which is recommended by Google.
 
-**Optional**:
-- [RxPermissions](https://github.com/tbruyelle/RxPermissions): Permissions powered by RxJava are regularly used, but not every project needs it.
-- [Bitrise](https://www.bitrise.io/): A CI/CD provider which is regularly used, but [GitHub Actions](https://github.com/features/actions) is our default choice.
-
 **Rejected**:
 - [Skeleton Layout](https://github.com/Faltenreich/SkeletonLayout): A progress indicator with visual feedback, it has a specific use case that is barely used.
 - [Mockito](https://github.com/mockito/mockito): A mocking framework which is barely used, because [Mockk](https://mockk.io/) is our default choice.
 
-Keep in mind, the features are based on _our team's_ requirements. In case the client has different requirements or requests, we can consider adding them as optional features if they occur regularly.
+Keep in mind, the features are based on _our team's_ requirements. In case the client has different requirements or requests, we can consider adding them as features if they occur regularly.
 
 Please note that the above examples are not definitive as new and existing libraries keep on emerging and evolving.
 
 Before an issue can be worked on, it must go through our voting process. 
 
-How do we vote on an issue? 🗳
+1. Create an issue 💭 
 
-- It is the responsibility of the RFC creator to label their proposed change as **essential** or **optional**.
-- If we agree with the RFC, we must react with 👍. 
-  - If there are 3 x 👍, then the issue is approved.
-- If we disagree with the RFC, we must react with 👎 and leave a comment explaining why. 
-  - If there are 3 x 👎, then the issue is rejected.
+- For features and chores, we must create an RFC with GitHub Discussions.
```

Comment:
Should we lower the tone here to `should` instead of `must`? Does that mean we always have to create RFC, even for the features or chores with obvious details or simple? 
- Say I would like to submit a ticket to upgrade the Kotlin or Gradle versions, should we vote for that? 
- Or when I would like to add more tests (not bringing anything new), will we have to wait for the voting process?

I'm wondering if this will slow down our contribution speed. I'm not trying against the RFC & voting process. However, we always have a process for the team to review & discuss the final approach in any contributions (pull requests).

Reply:
Using `should` instead of `must` would definitely give a friendlier message to readers, but I don't think using that word should be taken as an excuse to open any issues without prior voting process. Otherwise we should add details on how issues get voted on back to the readme and we're back to [this issue](https://github.com/nimblehq/android-templates/pull/409#discussion_r1115206580) again 🥴

Reply:
@luongvo Since we already made a vote on https://github.com/nimblehq/android-templates/discussions/417, we should proceed with this for now. 

If you don't agree, I think another RFC should be made to optimize the process better 🙏

Reply:
@Tuubz @ryan-conway Sorry for the question. However, the RFC mentioned **How `should` we suggest changes to the wiki?**, we did not say anything about a MUST or HAVE TO. So by using that word, I still have concern this process will slow down our contribution effort.

Reply:
@luongvo I understand your concern, but how do we decide if an issue is "simple" or "obvious"? 

This can be subjective depending on the developer's experience. 

Or... we make the exception for the repository maintainer to create issues without voting 💭

Reply:
@Tuubz @ryan-conway IMO, everyone in the chapter, in the company or external contributors, can submit new issues, questions, or RFCs. The repository maintainer decides 
- which submitted issue needs to be clarified & should go through the voting process with an RFC.
- which submitted issue is clear and fit to be selected for the next milestone.